app/src/components/map/wfs-utils.tsx

```diff
@@ -0,0 +1,269 @@
+import { Feature } from 'geojson';
+import React from 'react';
+import { ReProjector } from 'reproj-helper';
+import {
+  getInferredLayersInfoByProjectedGeometry,
+  getInferredLayersInfoByWFSFeature,
+  getLayerTypesToSkipByProjectedGeometry
+} from 'utils/mapLayersHelpers';
+import { defaultWFSParams, IWFSParams } from './WFSFeatureGroup';
+
+/**
+ * Alter the projection of an array of features, from EPSG:4326 to EPSG:3005 (BC Albers).
+ *
+ * @param {Feature[]} geos
+ * @return {*}  {Promise<any>[]}
+ */
+export const changeProjections = (geos: Feature[]): Promise<Feature>[] => {
+  const reprojector = new ReProjector();
+
+  return geos.map(
+    (geo: Feature) => reprojector.feature(geo).from('EPSG:4326').to('EPSG:3005').project() as Promise<Feature>
+  );
+};
+
+/**
+ * Generate the coordinates string for the reprojected geometries based on geometry type
+ *
+ * This is needed because the query for filtering results by geometry and layer(s) intersection
+ * is done using CQL_FILTER (https://docs.geoserver.org/master/en/user/services/wfs/vendor.html)
+ * and this function takes our projected geometry and converts it into a valid CQL-compatible coordinates string
+ *
+ * @param {Feature} projectedGeometry
+ * @returns {string} formatted coordinates string
+ *
+ */
+export const generateCoordinatesString = (projectedGeometry: any) => {
```

Comment:
Should this have an interface type? Received error on this function during testing.